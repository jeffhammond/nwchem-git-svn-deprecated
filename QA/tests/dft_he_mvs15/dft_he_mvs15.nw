start dft_he_mvs15

geometry 
 he 0 0 0
 symmetry oh
end

basis
 * library cc-pvtz
end

dft
 xc xmvs15
end

task dft