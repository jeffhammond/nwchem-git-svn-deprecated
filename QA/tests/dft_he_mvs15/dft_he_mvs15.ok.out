 argument  1 = dft_he_mvs15.nw
                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 6.5
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                              Copyright (c) 1994-2013
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute
 
             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file
 
                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = lagrange
    program         = /home/edo/nwchem/bin/LINUX64/nwchem
    date            = Mon Oct  5 18:11:55 2015

    compiled        = Fri_Sep_25_12:58:00_2015
    source          = /home/edo/nwchem
    nwchem branch   = Development
    nwchem revision = 27477
    ga revision     = 10586
    input           = dft_he_mvs15.nw
    prefix          = dft_he_mvs15.
    data base       = ./dft_he_mvs15.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107198 doubles =    100.0 Mbytes
    stack    =   13107195 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428793 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------
 
  0 permanent = .
  0 scratch   = .
 
 
 
 
                                NWChem Input Module
                                -------------------
 
 

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 
 
                             Geometry "geometry" -> ""
                             -------------------------
 
 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)
 
  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 he                   2.0000     0.00000000     0.00000000     0.00000000
 
      Atomic Mass 
      ----------- 
 
      he                 4.002600
 

 Effective nuclear repulsion energy (a.u.)       0.0000000000

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000
 
      Symmetry information
      --------------------
 
 Group name              Oh       
 Group number             44
 Group order              48
 No. of unique centers     1
 
      Symmetry unique atoms
 
     1
 
 
            XYZ format geometry
            -------------------
     1
 geometry
 he                    0.00000000     0.00000000     0.00000000
 


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          cc-pvtz                   on all atoms 


  xc_inp: hfexch multiplicative factor not found.
 
                                 NWChem DFT Module
                                 -----------------
 
 
                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  he (Helium)
  -----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  2.34000000E+02  0.002587
  1 S  3.51600000E+01  0.019533
  1 S  7.98900000E+00  0.090998
  1 S  2.21200000E+00  0.272050
 
  2 S  6.66900000E-01  1.000000
 
  3 S  2.08900000E-01  1.000000
 
  4 P  3.04400000E+00  1.000000
 
  5 P  7.58000000E-01  1.000000
 
  6 D  1.96500000E+00  1.000000
 


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 he                         cc-pvtz                  6       15   3s2p1d




 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 he                         cc-pvtz                  6       15   3s2p1d


      Symmetry analysis of basis
      --------------------------
 
        a1g         4
        a1u         0
        a2g         0
        a2u         0
        eg          2
        eu          0
        t1g         0
        t1u         6
        t2g         3
        t2u         0
 
  Caching 1-el integrals 
 
            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     1
          No. of electrons :     2
           Alpha electrons :     1
            Beta electrons :     1
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: on ; symmetry adaption is: on 
          Maximum number of iterations:  30
          AO basis - number of functions:    15
                     number of shells:     6
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04
 
              XC Information
              --------------
                     Hartree-Fock (Exact) Exchange  1.000          
 
          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 

 
      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08

 
      Superposition of Atomic Density Guess
      -------------------------------------
 
 Sum of atomic energies:          -2.86115334
 
      Non-variational initial energy
      ------------------------------

 Total energy =      -2.861153
 1-e energy   =      -3.887056
 2-e energy   =       1.025903
 HOMO         =      -0.917625
 LUMO         =       0.590162
 
 
      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------
 

 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated

  Symmetry fudging

 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated

  Numbering of irreducible representations: 
 
     1 a1g         2 a1u         3 a2g         4 a2u         5 eg      
     6 eu          7 t1g         8 t1u         9 t2g        10 t2u     
 
  Orbital symmetries:
 
     1 a1g         2 a1g         3 t1u         4 t1u         5 t1u     
     6 a1g         7 t2g         8 t2g         9 t2g        10 eg      
    11 eg      
 
   Time after variat. SCF:      0.1
   Time prior to 1st pass:      0.1

 #quartets = 2.310D+02 #integrals = 1.797D+03 #direct =  0.0% #cached =100.0%


 Integral file          = ./dft_he_mvs15.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = 403470
 No. of bits per label  =      8        No. of bits per value  =     64


 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated

  Symmetry fudging

 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated


           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       12.98            12975126
          Stack Space remaining (MW):       13.11            13106988

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1     -2.8611535721 -2.86D+00  1.22D-05  3.92D-08     0.1

 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated

  Symmetry fudging

 !! scf_movecs_sym_adapt:    5 vectors were symmetry contaminated

 d= 0,ls=0.0,diis     2     -2.8611535740 -1.90D-09  1.50D-06  3.28D-10     0.1


         Total DFT energy =       -2.861153574003
      One electron energy =       -3.887047706479
           Coulomb energy =        2.051788264953
    Exchange-Corr. energy =       -1.025894132476
 Nuclear repulsion energy =        0.000000000000

 Numeric. integr. density =        0.000000000000

     Total iterative time =      0.0s


 
                  Occupations of the irreducible representations
                  ----------------------------------------------
 
                     irrep           alpha         beta
                     --------     --------     --------
                     a1g               1.0          1.0
                     a1u               0.0          0.0
                     a2g               0.0          0.0
                     a2u               0.0          0.0
                     eg                0.0          0.0
                     eu                0.0          0.0
                     t1g               0.0          0.0
                     t1u               0.0          0.0
                     t2g               0.0          0.0
                     t2u               0.0          0.0
 
 
                       DFT Final Molecular Orbital Analysis
                       ------------------------------------
 
 Vector    1  Occ=2.000000D+00  E=-9.176298D-01  Symmetry=a1g
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.477199  1 He s                  1      0.354984  1 He s          
     3      0.308050  1 He s          
 
 Vector    2  Occ=0.000000D+00  E= 5.901538D-01  Symmetry=a1g
              MO Center= -1.3D-16,  2.3D-16,  4.0D-17, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.731365  1 He s                  3     -1.698091  1 He s          
    10     -0.156177  1 He dxx               13     -0.156177  1 He dyy        
    15     -0.156177  1 He dzz        
 
 Vector    3  Occ=0.000000D+00  E= 1.500510D+00  Symmetry=t1u
              MO Center=  4.5D-19,  1.3D-18, -7.7D-18, r^2= 4.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      1.018762  1 He pz                 8     -0.173928  1 He py         
 
 Vector    4  Occ=0.000000D+00  E= 1.500510D+00  Symmetry=t1u
              MO Center=  4.4D-17, -2.5D-16, -4.0D-17, r^2= 4.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.006638  1 He py                 7     -0.179553  1 He px         
     9      0.161438  1 He pz         
 
 Vector    5  Occ=0.000000D+00  E= 1.500510D+00  Symmetry=t1u
              MO Center=  8.9D-17,  1.5D-17,  7.7D-18, r^2= 4.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.017785  1 He px                 8      0.167483  1 He py         
 
 Vector    6  Occ=0.000000D+00  E= 3.441766D+00  Symmetry=a1g
              MO Center=  1.6D-25,  3.9D-24,  3.2D-24, r^2= 8.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      4.168373  1 He s                  1     -1.724924  1 He s          
     3     -1.308853  1 He s                 10     -0.822471  1 He dxx        
    13     -0.822471  1 He dyy               15     -0.822471  1 He dzz        
 
 Vector    7  Occ=0.000000D+00  E= 6.357598D+00  Symmetry=t2g
              MO Center=  2.4D-34,  7.1D-34,  1.4D-33, r^2= 2.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    12      1.527319  1 He dxz               11      0.792215  1 He dxy        
    14     -0.199227  1 He dyz        
 
 Vector    8  Occ=0.000000D+00  E= 6.357598D+00  Symmetry=t2g
              MO Center= -1.8D-34, -1.6D-33,  1.4D-34, r^2= 2.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    14      1.415546  1 He dyz               11      0.949456  1 He dxy        
    12     -0.307832  1 He dxz        
 
 Vector    9  Occ=0.000000D+00  E= 6.357598D+00  Symmetry=t2g
              MO Center=  7.4D-35,  6.2D-34, -5.2D-34, r^2= 2.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11      1.212818  1 He dxy               14     -0.978027  1 He dyz        
    12     -0.756660  1 He dxz        
 
 Vector   10  Occ=0.000000D+00  E= 6.357598D+00  Symmetry=eg
              MO Center= -2.7D-33, -4.9D-33, -5.6D-35, r^2= 2.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    10      0.890208  1 He dxx               13     -0.839626  1 He dyy        
 
 Vector   11  Occ=0.000000D+00  E= 6.357598D+00  Symmetry=eg
              MO Center=  7.8D-34,  1.3D-33, -3.3D-34, r^2= 2.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      0.998720  1 He dzz               13     -0.543165  1 He dyy        
    10     -0.455555  1 He dxx        
 

 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.0s     wall:        0.2s
 
 
                                NWChem Input Module
                                -------------------
 
 
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  121      121     3160     1200      622        0        0       94     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             7.96e+05 2.67e+05 1.12e+06 0.00e+00 0.00e+00 7.52e+02
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 70200 bytes
 
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        22	        19
	current total bytes		         0	         0
	maximum total bytes		   1056584	  22510264
	maximum total K-bytes		      1057	     22511
	maximum total M-bytes		         2	        23
 
 
                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:
 
                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018
 
                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
        S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen, V. Anisimov,
    F. Aquino, S. Hirata, M. T. Hackler, T. Risthaus, M. Malagoli, A. Marenich,
   A. Otero-de-la-Roza, J. Mullin, P. Nichols, R. Peverati, J. Pittner, Y. Zhao,
          P.-D. Fan, A. Fonari, M. Williamson, R. J. Harrison, M. Dupuis,
     D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju, M. Krishnan,
       B. E. Van Kuiken, A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu,
   T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros,
     G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols,
       K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski,
       T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood,
      E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju,
        R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:        0.1s     wall:        0.3s
